ceph.spec.in

```diff
@@ -2437,6 +2437,7 @@ fi
 
 %files -n cephfs-top
 %{python3_sitelib}/cephfs_top-*.egg-info
+%{_bindir}/fstopbase.py
```

Comment:
@joscollin I think this should be

```
%{python3_sitelib}/fstopbase.py
```

Could you try this along with the other change for debian build.

Reply:
okay, will check that.

Reply:
created a new ticket to work on this issue: https://tracker.ceph.com/issues/58561